src/Core/3DTiles/C3DTFeature.js

```diff
@@ -8,19 +9,17 @@ import { Object3D, Box3 } from 'three';
  * @param {number} tileId - tileId
  * @param {number} batchId - batch id
  * @param {Array<{start:number,count:number}>} groups - groups in geometry.attributes matching batchId
- * @param {object} info - info in the batchTable
  * @param {object} [userData] - some userData
  * @param {Object3D} object3d - object3d in which feature is present
  * @property {number} tileId - tile id
  * @property {Object3D} object3d - object3d in which feature is present
  * @property {number} batchId - batch id
  * @property {Array<{start:number,count:number}>} groups - groups in geometry.attributes matching batchId
- * @property {object} info - info in the batchTable
  * @property {object} [userData] - some userData
  */
 class C3DTFeature {
     #info;
-    constructor(tileId, batchId, groups, info, userData, object3d) {
+    constructor(tileId, batchId, groups, userData, object3d) {
```

Comment:
This is some breaking change since `C3DTFeature` is exposed to the end-user.
However, is there anyone which instantiates `C3DTFeature` directly? Why it is exposed then?

Reply:
Yes it is but I had the same interrogations and I think the answers are nobody instantiates it directly and it shouldn't be exposed (I looked into LIRIS's code which are the main users of this and haven't seen any instantiation of `C3DTFeature`). We should remove it from the public API but I guess this is the purpose of another PR. Meanwhile I think we should accept having this breaking change and document it in the commit (and therefore the release), but no strong opinions on this, we can also do it differently. WDYT ?

Reply:
Agreed with the breaking change! We should keep C3DTFeature exposed for now, it may be the object of another PR.